### ISSUE TITLE
vf-analytics-google javascript mispelling

### ISSUE THREAD

**Issue by nbuso** [visual-framework/vf-core#1118]
In the GA tracker javascript code there is a misspelling on the function "vfGaLinkTrackingInit"
```javascript
function vfGaLinkTrackingInit() {
  // jQuery("body.google-analytics-loaded .track-with-analytics-events a").on('mousedown', function(e) {
  //   analyticsTrackInteraction(e.target,'Manually tracked area');
  // });

  document.body.addEventListener("mousedown", function (evt) {

    // send GA events if GA closest area is detected
    let closestContainer = getClosestGa(evt.target, '[data-vf-google-anlaytics-region]');
    if(closestContainer) {
      analyticsTrackInteraction(evt.target);
    }
    else {
```

the region attribute the code is looking for should be 'data-vf-google-analytics-region'

**Comment by khawkins98** [comment]
This was fixed in 1.0.0-rc.1 https://github.com/visual-framework/vf-core/pull/1066/files#diff-fb3bff43cf5b193e1ee8aea356bf72e4R4

Are you perhaps using an older version from npm or older tagged release?

**Comment by khawkins98** [comment]
I think this was previously resolved, please reopen if needed.
